src/components/blipTable/blipTable.component.ts

```diff
@@ -0,0 +1,133 @@
+import angular from 'core/angular';
```

Comment:
Is this import correct? Maybe `import angular from 'angular'` like other angular imports? 🤔